### PR TITLE
Colorizer - Vuetify v2

### PR DESCRIPTION
**Pull request by SoldoV**
Changed colorizer to vuetify v2
Issue on builder : https://github.com/nsftx/chameleon-builder/issues/491
Connected with PR on builder :  https://github.com/nsftx/chameleon-builder/pull/500

### CODE REVIEW

**Comment by Lanchi** [comment]
Actually, since Vuetify v2 is still in beta phase, let's not merge this to master for now. I published `1.0.17-beta.1` version to npm so we can use it in Builder.

**Comment by Sneki** [comment]
When I run this component locally:

image

File: src/components/VColorPaletteMaterial.vue [hunk 1]
```diff
@@ -63,6 +67,9 @@ export default {
     value() {
       this.setColorFromInput();
     },
+    hexColor() {
```
Comment (Sneki) [inline src/components/VColorPaletteMaterial.vue]: You can achieve same functionality without watcher, with computed property. Set computed value hexNum as: _return this.hexColor ? this.hexColor.value : null;_

File: src/components/VColorPaletteMaterial.vue [hunk 2]
```diff
@@ -219,6 +225,7 @@ export default {
 
   .text-field-hex {
     padding: 0px !important;
+    color:black !important;
```
Comment (Sneki) [inline src/components/VColorPaletteMaterial.vue]: I've noticed few formatting issues, like this one. Add spacing between color and its value.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nsoft/vuetify-colorizer",
-  "version": "1.0.17-beta.2",
+  "version": "1.0.17-beta.3",
```
Comment (Lanchi) [inline package.json]: Please keep beta tags with `beta` prefix when publishing
[Tags](https://www.npmjs.com/package/@nsoft/vuetify-colorizer)

File: src/components/VColorPaletteMaterial.vue [hunk 4]
```diff
@@ -258,12 +269,7 @@ export default {
     height: 100% !important;
```
Comment (Sneki) [inline src/components/VColorPaletteMaterial.vue]: Remove unnecessary css (replace with this): 
 .color-group-wrap {
  display: flex;
}
.color-group {
  display: flex;
  flex-direction: column;
}

File: src/components/VColorPaletteMaterial.vue [hunk 5]
```diff
@@ -288,5 +294,4 @@ export default {
     }
   }
 }
-
```
Comment (Sneki) [inline src/components/VColorPaletteMaterial.vue]: Do we need this?

 **&-base** {
      width: 40px;
    }
    **&-light** {
      color: black;
    }

**Pull request merged**